gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/CoopLockFsckRunner.java

```diff
@@ -363,6 +347,60 @@ private void deleteAndRenameToRepairRenameOperation(
     return Optional.empty();
   }
 
+  private boolean unlockOperationIfNoLogUnchecked(
+      Map.Entry<FileStatus, CoopLockRecord> operation) {
+    try {
+      return unlockOperationIfNoLog(operation);
+    } catch (IOException e) {
+      throw new RuntimeException(
+          String.format("Failed to check if %s operation expired", operation.getValue()), e);
+    }
+  }
+
+  private boolean unlockOperationIfNoLog(Map.Entry<FileStatus, CoopLockRecord> operation)
+      throws IOException {
+    CoopLockRecord operationRecord = operation.getValue();
+    String globPath =
+        CoopLockRecordsDao.LOCK_DIRECTORY + "*" + operationRecord.getOperationId() + "*.log";
+    URI globUri =
+        gcsFs.getPathCodec().getPath(bucketName, globPath, /* allowEmptyObjectName= */ false);
+    FileStatus[] operationLocks = ghfs.globStatus(new Path(globUri));
+    checkState(
+        operationLocks.length < 2,
+        "operation %s should not have more than one log file",
+        operationRecord.getOperationId());
+
+    // Log file not created - nothing to repair
+    if (operationLocks.length == 0) {
```

Comment:
Do we wanna return false too if multiple logs found?

operationLocks looks like a hard-to-read name also since how it implies # of log files anyway.

Reply:
No, multiple log files mean that there are something wrong and we should fail whole thing.

Fixed name.